### PR TITLE
Z13/People

### PR DESCRIPTION
**Pull request by szymonbasiul**
Prepared people tiles, API implementation, searching feature. Small styling changes with image/name elements coming soon.

### CODE REVIEW

**Review by AndriiStafiniak — APPROVED** [review]
It looks great. Good job 😎 just a few small things to improve

**Comment by AndriiStafiniak** [comment]
but this branch has  conflicts, how could you solve them and we can merge 😉

File: src/components/common/Tile/PersonTile/PersonTile.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            src={`https://image.tmdb.org/t/p/original${member.profile_path}`}
+            alt=""
+          />
+        ) : (
+          <StyledPictureIcon />
+        )}
+      </ImageWrapper>
+      <Character>
+        <Name>{member.name}</Name>
+      </Character>
```
Comment (AndriiStafiniak) [inline src/components/common/Tile/PersonTile/PersonTile.jsx:19]: here is a (p) element inside another (p) element. We can't do this. please correct it.

Comment (jakub-bartoszek) [reply src/components/common/Tile/PersonTile/PersonTile.jsx:19]: As Andrii said, you need to change that

File: src/components/common/Tile/PersonTile/styled.jsx
Comment (jakub-bartoszek) [inline src/components/common/Tile/PersonTile/styled.jsx:1]: If name of the person is too long, it doesnt look good. 
image

**Review by jakub-bartoszek — CHANGES_REQUESTED** [review]
It is worth to add "status" of fetching people in your slice. When you do that, you can add conditional rendering of the People page like i did in Movies page. :)

File: src/components/common/Tile/PersonTile/PersonTile.jsx (hunk 1 above)
Comment (szymonbasiul) [reply src/components/common/Tile/PersonTile/PersonTile.jsx:19]: Thanks for feed. Updated

**Comment by szymonbasiul** [comment]
Updates added.

**Comment by jakub-bartoszek** [comment]
Perfect!

**Pull request merged**